### DISCUSSION TITLE
How to plug in the new 'tissue-specific ontogeny?

### DISCUSSION THREAD

**Discussion by misol** [Open-Systems-Pharmacology/Forum#1521] [Questions & Problems] [answered]
Hello there,

I am a beginner in PK-Sim software.

I have a question about how to implement tissue-specific ontogeny, especially transporters.

According to [the PK-Sim manual](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-profile#settings-in-the-protein-expression-tab), I can import an ontogeny file for liver and intestine.
Can I import ontogeny data for the liver and kidney transporters separately?

For example, the preloaded enzyme CYP3A4 has two tissue-specific ontogenies like below:
![image](https://user-images.githubusercontent.com/3437916/236052247-a9b979d1-24a9-4c7d-a1cc-86a8080b8dca.png)

However, I could not find ways to import tissue-specific ontogeny for my own transporters. I could import my sample Excel file as below, but I don't know how to specify the tissue or organ on the file.

I tried to specify the target organ in a column, but it seemed to be not working.
![image](https://user-images.githubusercontent.com/3437916/236053967-e3443179-9a94-4644-aa84-e6a51463884d.png)

For the test data with meaningless values (not real expression data), I used this Excel file [test.xlsx](https://github.com/Open-Systems-Pharmacology/Forum/files/11391386/test.xlsx)
.
![image](https://user-images.githubusercontent.com/3437916/236054154-8ade2c69-33d4-4788-862a-0bdbd323b680.png)

The specific data for ontogeny, what I would like to use, is below:

- Hepatic transporters: B D van Groen et al, Pharmacol Rev. 2021 https://doi.org/10.1124/pharmrev.120.000071 (might need to extract the value from the figures)
- Renal transporters: Cheung et al., CPT 2019 https://doi.org/10.1002/cpt.1516 (There are specific values on the supplementary)

And FYI, B D van Groen et al wrote a discussion below:

> Moreover, although this review is limited to ontogeny in the liver, DTs and/or DMEs are also abundant in other major organs, such as the kidney and gastrointestinal tract, and sanctuary sites, including the central nervous system (DeGorter et al., 2012). Developmental patterns of isoforms appear to be organ-dependent (Drozdzik et al., 2018; Li et al., 2019). However, for other organs, a quantitative approach as presented in this comprehensive review is not available but is highly needed. Once this is available, integration of ontogeny profiles in multiple tissues via a PBPK framework could provide a more holistic systems approach on the development of an entire organism (Smits et al., 2013).

Thank you for this open project!

**Comment by prvmalik** [comment, marked as answer]
Consider separating the enzyme and its expression into two (or more). In other words add UGT1a9 (kidney only) and UGT1a9 (everything but kidney), for example.

**Reply by msevestre** [reply]
Agreed. This is the only way to achieve what you want I believe

**Reply by misol** [reply]
Thank you both!

The issue may be solved. I will try that way.

Sorry for some typo, I find them right now.

In that method, if I wanted to predict transporter-mediated drug-drug interaction assuming this drug is a victim drug, the Ki for the perpetrator drug should also be repeated.

Okay!
